tests/41end-to-end-keys/07-backup.pl

```diff
@@ -393,9 +435,19 @@ =head2 matrix_get_backup_key
 sub matrix_get_backup_key {
    my ( $user, $room_id, $session_id, $version ) = @_;
 
+   my $uri;
+
+   if ($session_id) {
```

Comment:
probably better to check for definedness rather than truthiness (`if ( defined $session_id )`) to distinguish between an absent param and an empty one.

Also, spaces inside parens please. (I don't make the rules...)